src/main/java/me/karakelley/tictactoe/Main.java

```diff
@@ -3,6 +3,7 @@
 public class Main {
 
     public static void main(String args[]) {
-        System.out.println("Hello!");
+            UI userInterface = new UI();
```

Comment:
Is the indentation off here? There's a way you can have IntelliJ clean up your code for you.